public/js/selfoss-events-navigation.js

```diff
@@ -173,12 +170,13 @@ selfoss.events.navigation = function() {
                     $('#nav-mobile-settings').click();
                     
                 // refresh list
-                 selfoss.reloadList();
+                selfoss.sync(true);
```

Comment:
Good point. I got used to waiting for the page to reload from previous versions but this might be actually better. In the future, it would be nice to add UI for reloading the list.